Array/src/ArrayTask4.java

```diff
@@ -0,0 +1,86 @@
+import java.util.Scanner;
+
+public class ArrayTask4 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String userAction;
+        String enterWords;
+        String[] words;
+        String enterValue;
+        int enterInt;
+        boolean end = false;
+        int findValue;
+        do {
+            System.out.println("Enter 3 or more words separated by commas without gaps");
+            enterWords = sc.nextLine();
+            words = enterWords.split(",");
+            if (!enterWords.contains(",") || words.length < 3) {
+                System.out.println("Something wrong! You should enter 3 or more words separated by commas without gaps ");
+            } else {
+                end = true;
+            }
+        } while (!end);
+        do {
+            System.out.println("Choose an action with an array of words and enter a number: ");
+            System.out.println("1 - maximum length ");
+            System.out.println("2 - start with");
+            System.out.println("3 - end with");
+            System.out.println("4 - contains");
+            System.out.println("0 - exit");
+            userAction = sc.next();
+            findValue = 0;
```

Comment:
Please initialize it in the same place where declare

Reply:
I can't initialize "findValue" where it's declared, because If I do, My "findValue" variable logic will not work correctly: it will not have zero value after the first successful match, and then for other operations, it won't be zero. 
At the moment My program works correctly: "findValue" variable is always zero before the operation begins.

Reply:
ok